examples/basic.go

```diff
@@ -16,6 +16,10 @@ func main() {
 		log.Fatal(err)
 	}
 
+	// how to set pool size
+	t38c.New(":9851", t38c.Debug, t38c.SetPoolSize(10))
```

Comment:
I think it would be better to delete this line, and add ```t38c.SetPoolSize(10)``` as the client initialization param above.

Reply:
Actually I'd like to refactor the whole configuration section, to pass an instance of a struct (ConfigParam) to the `New` function like other databases but since I'm not fully aware of server configurations I haven't done it yet.

Reply:
Hmm why you dont liked this configuration style?

Reply:
In my view, ConfigParam gathered everything in one place and developer has to fill it once without multiple functions call or variable declarations.